core/server/master/src/test/java/alluxio/master/file/PermissionCheckerTest.java

```diff
@@ -313,15 +313,6 @@ public void selfCheckFailBySameGroup() throws Exception {
     checkPermission(TEST_USER_3, Mode.Bits.WRITE, TEST_DIR_FILE_URI);
   }
 
-  @Test
-  public void checkFallThrough() throws Exception {
```

Comment:
Should this test stay, but check for the expected behavior?

Reply:
added